### PR TITLE
feat(replay): Update docs for starting/resuming recording

### PR DESCRIPTION
**Pull request by billyvg**
This adds docs for lazy loading Replay and fixes the docs for start/stop recording.

### CODE REVIEW

**Comment by billyvg** [comment]
Note I changed the language around a bit from sentry-javascript

File: src/platform-includes/session-replay/setup/javascript.mdx [hunk 1]
```diff
@@ -53,3 +53,21 @@ You can verify that with the following:
...
+
+### Lazy loading Replay
+
+Replay will start automatically when you add the integration.
+If you do not want to start Replay immediately (e.g. if you want to lazy-load it),
+you can also use `addIntegration` to load it later:
+
+```js
+Sentry.init({
+  // Do not load it initially
```
Comment (bruno-garcia) [inline src/platform-includes/session-replay/setup/javascript.mdx:65]:
```suggestion
  // Note Replay is NOT instantiated below:
```

**Review by bruno-garcia — APPROVED** [review]

File: src/platform-includes/session-replay/setup/javascript.mdx [hunk 2]
```diff
@@ -53,3 +53,21 @@ You can verify that with the following:
 ```javascript
 myUndefinedFunction();
 ```
+
+### Lazy loading Replay
```
Comment (lizokm) [inline src/platform-includes/session-replay/setup/javascript.mdx:57]:
```suggestion
### Lazy-loading Replay
```

File: src/platform-includes/session-replay/setup/javascript.mdx [hunk 3]
```diff
@@ -53,3 +53,21 @@ You can verify that with the following:
...
+
+Replay will start automatically when you add the integration.
+If you do not want to start Replay immediately (e.g. if you want to lazy-load it),
+you can also use `addIntegration` to load it later:
```
Comment (lizokm) [inline src/platform-includes/session-replay/setup/javascript.mdx:61]:
```suggestion
Once you've added the integration, Replay will start automatically. If you don't want to start it immediately (lazy-load it), you can use `addIntegration`:
```

File: src/platforms/javascript/common/session-replay/configuration.mdx [hunk 4]
```diff
@@ -56,22 +56,38 @@ new Replay({
 
 ## Start and Stop Recording
 
-Replay recording only starts automatically when it's included in the integrations key when calling `Sentry.init`. Otherwise, you can initialize the plugin and manually call the `start()` method on the integration instance. To stop recording you can call the `stop()`:
+Replay recording starts automatically when it's included in the `integrations` key when calling `Sentry.init()` or when adding it via `addIntegration()`. Call `stop()` to stop recording and `start()` to resume recording:
```
Comment (lizokm) [inline src/platforms/javascript/common/session-replay/configuration.mdx:59]:
```suggestion
Replay recording starts automatically if it's included in the `integrations` key when calling `Sentry.init()` or added via `addIntegration()`. To stop recording, call `stop()`; to resume, call `start()`:
```

File: src/platforms/javascript/common/session-replay/configuration.mdx [hunk 5]
```diff
@@ -56,22 +56,38 @@ new Replay({
...
+// Load replay when you configure Sentry SDK
+Sentry.init({
+  integrations: [replay]
+});
+
+// Or if you want to defer loading Replay
+getCurrentHub().getClient().addIntegration(replay);
 
-replay.stop(); // Stop recording
+// some time later
```
Comment (lizokm) [inline src/platforms/javascript/common/session-replay/configuration.mdx:72]:
```suggestion
// sometime later
```

File: src/platforms/javascript/common/session-replay/configuration.mdx [hunk 6]
```diff
@@ -56,22 +56,38 @@ new Replay({
...
+Sentry.init({
+  integrations: [replay]
+});
 
-replay.start(); // Start recording
+// Or if you want to defer loading Replay
+Sentry.getCurrentHub().getClient().addIntegration(replay);
 
-replay.stop(); // Stop recording
+// some time later
```
Comment (lizokm) [inline src/platforms/javascript/common/session-replay/configuration.mdx:88]:
```suggestion
// sometime later
```

**Review by lizokm — APPROVED** [review]
Few small tweaks, otherwise LGTM

File: src/platforms/javascript/common/session-replay/configuration.mdx (hunk 4 above)
Comment (Lms24) [reply src/platforms/javascript/common/session-replay/configuration.mdx:59]: super-l: In addition to Liza's suggestion, let's maybe replace key with array? 

```suggestion
Replay recording starts automatically if it's included in the `integrations` array when calling `Sentry.init()` or added via `addIntegration()`. To stop recording, call `stop()`; to resume, call `start()`:

**Review by Lms24 — APPROVED** [review]

**Comment by billyvg** [comment]
Thanks everyone!

**Pull request merged**